graspy/inference/latent_distribution_test.py

```diff
@@ -93,6 +95,40 @@ class LatentDistributionTest(BaseInference):
         power or less power under which alternatives. Generally not recomended,
         as it is untested and included for experimental purposes.
 
+    align_type : str, {'sign_flips' (default), 'seedless_procrustes'} or None
+        Random dot product graphs have an inherent non-identifiability,
+        associated with their latent positions. Thus, two embeddings of
+        different graphs may not be orthogonally aligned. Without this accounted
+        for, two embeddings of different graphs may appear different, even
+        if the distributions of the true latent positions are the same.
+        There are several options in terms of how this can be addresssed:
+
+        - 'sign_flips'
+            A simple heuristic that flips the signs of one of the embeddings,
+            if the medians of the two embeddings in that dimension differ from
+            each other. See :class:`~graspy.align.SignFlips` for more
+            information on this procedure. In the limit, this is guaranteed to
+            lead to a valid test, as long as matrix :math:`X^T X`, where
+            :math:`X` is the latent positions does not have repeated non-zero
+            eigenvalues. This may, however, result in an invalid test in the
+            finite sample case if the some eigenvalues are same or close.
+        - 'seedless_procrustes'
+            An algorithm that learns an orthogonal alignment matrix. This
+            procedure is slower than sign flips, but is guaranteed to yield a
+            valid test in the limit, and also makes the test more valid in some
+            finite sample cases, in which the eigenvalues are very close to
+            each other. See `~graspy.align.SignFlips` for more information on
+            the procedure.
+        - 'none'
```

Comment:
string "Nones" can lead to confusion IMO. Thoughts on making this an actual `None`?

Reply:
or maybe @dwaynepryce or @j1c  can tell me if this is nitpicky?

Reply:
oh i see you already support `None` so maybe just remove string 'none'?

Reply:
i think this was just a docstring issue. fxd.